### ISSUE TITLE
No instructions on how to use the model once it is generated..

### ISSUE THREAD

**Issue by azamsharp** [apple/turicreate#3059]
There are no instructions on how to use the model once it has been generated. Please include a sample app for Activity Classification app.

**Comment by TobyRoseman** [comment]
@azamsharp - please clarify what exactly you want. Are you just looking for example code to get predictions from an activity classification model? Or are you looking for a full iOS example app that uses an activity classification model?

**Comment by azamsharp** [comment]
Full iOS app

**Comment by TobyRoseman** [comment]
I agree this would be nice, particularly for the activity classifier.

The first step to using an activity classifier in a iOS app would be to export it to Core ML and integrate it in your app. We do have [a page about doing that](https://apple.github.io/turicreate/docs/userguide/activity_classifier/export_coreml.html) in our user guide. Also there is an [iOS example app for using a Core ML model](https://developer.apple.com/documentation/vision/classifying_images_with_vision_and_core_ml), it uses an image classifier model not an activity classification model.
